CONTRIBUTING.md

```diff
@@ -1,344 +1,508 @@
+differently.
 
-The following outlines how BIDS developers think about communicating through issues.
+The following outlines how BIDS developers think about communicating through
+issues.
 
-**Issues** are individual pieces of work that need to be completed or decisions that need to be made to move the project forwards.
-A general guideline: if you find yourself tempted to write a great big issue that
-is difficult to describe as one unit of work, please consider splitting it into two or more issues.
+**Issues** are individual pieces of work that need to be completed or decisions
+that need to be made to move the project forwards. 
+A general guideline: if you find yourself tempted to write a great big issue 
+that is difficult to describe as one unit of work, please consider splitting it 
+into two or more issues.
 
-Issues are assigned [labels](#issue-labels) which explain how they relate to the overall project's goals and immediate next steps.
+Issues are assigned [labels](#issue-labels) which explain how they relate to the
+overall project's goals and immediate next steps.
 
 ### Issue labels
 
-The current list of labels are [here](https://github.com/bids-standard/bids-specification/labels) and include:
-
-* [![Help wanted](https://img.shields.io/badge/-help%20wanted-159818.svg)](https://github.com/bids-standard/bids-specification/labels/community) *These issues contain a task that a member of the team has determined we need additional help with.*
-
-    If you feel that you can contribute to one of these issues, we especially encourage you to do so!
-
-* [![Opinions wanted](https://img.shields.io/badge/-opinions%20wanted-84b6eb.svg)](https://github.com/bids-standard/bids-specification/labels/opinions%20wanted) *These issues hold discussions where we're especially eager for feedback.*
-
-    Ongoing discussions benefit from broad feedback.
-    This label is used to highlight issues where decisions are being considered, so please join the conversation!
-
-* [![Community](https://img.shields.io/badge/-community-%23ddcc5f.svg)](https://github.com/bids-standard/bids-specification/labels/community) *These issues are related to building and supporting the BIDS community.*
-
-    In addition to the specification itself, we are dedicated to creating a healthy community.
-    These issues highlight pieces of work or discussions around how we can support our members and make it easier to contribute.
-
-## Commenting on a pull request
-
-Our primary method of adding to or enhancing BIDS occurs in the form of [pull requests](https://help.github.com/articles/about-pull-requests/).
-BIDS Extension Proposals ([BEPs](https://docs.google.com/document/d/1pWmEEY-1-WuwBPNy5tDAxVJYQ9Een4hZJM06tQZg8X4/)) are submitted as pull requests, and commenting on pull requests is an important way of participating in the BIDS community.
+The current list of labels are
+[here](https://github.com/bids-standard/bids-specification/labels) and include:
```

Comment:
Reading through those again to reuse some of that document for the bids-matlab repo, it makes me realize:

Does it make sense to have a couple more issue labels added here?

I think that some of our labels are self-explanatory but others might not be. Any thoughts one that?

Reply:
I don't feel strongly about *which* labels are listed, but I think that (1) not every label should be listed, otherwise it becomes too long of a list that readers may skip, and (2) the information should suffice that users check out the labels and their descriptions themselves.

And we are responsible to make labels that are easy to understand and that have this little description that GitHub allows to attach to each label.

Reply:
Makes sense to me. Let's leave it like that for now then.